tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -265,13 +265,169 @@ Ramdump received successfully..!
 DumpParser Script provides two interfaces: CUI and GUI
 
 ### DumpParser using CUI
+
+#### To get call stack using RAM dump
 1. Run Ramdump Parser Script
 ```
 cd $TIZENRT_BASEDIR/tools/dump_tool/
 python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
 2. See the Output
 
+#### To display Debug Symbols/Crash point using assert logs
+1. Copy crash logs
+    First copy the crash logs to a file named log_file in tools/dump_tool/`<log_file>`
+2. Run Script in tools/dump_tool path
+    $ python3 ramdumpParser.py -e `<Elf path>` -t `<Log file path>`
```

Comment:
Can you please add some more information here regarding what elf should be given? As I can see the app and common elf paths are generated using the app names that were fetched from the log file but still in the below few examples wifi_dbg and common_dbg were given as an argument.

Reply:
This elf file information is for the rest of the dump parser functionality.
Getting call stack for cortex R devices, if below criteria is  is FALSE:
_if ('CONFIG_ARCH_FAMILY="armv8-m"' in data) or ('CONFIG_ARCH_FAMILY="armv7-m"' in data):_

But the exact crash point & debug display symbols do not need this information elf information just as you pointe dout, they are being fetched using the dynamic apps from the dump_logs